src/handlers/userStore.ts

```diff
@@ -0,0 +1,124 @@
+import { Subscription } from "nats";
+
+import { jsonCodec, NatsHandler } from "../nats";
+import { getClient } from "../elasticSearch";
+import { ELASTICSEARCH_INDEX_NAME } from "../config";
+import { JSONUser } from "../user";
+
+export const userStoreHandlers: NatsHandler[] = [
+    [
+        "item-store_add_user",
+        async (subscription: Subscription): Promise<void> => {
+            const client = getClient();
+
+            for await (const message of subscription) {
+                const user = jsonCodec.decode(message.data) as JSONUser;
+
+                try {
+                    const reponse = await client.index({
```

Comment:
let's move the logic  for creating/getting/updating a user document to a `userStore` service in `services` like we do with inventory and warehouse